samples/will_it_shuffle.lobster

```diff
@@ -0,0 +1,228 @@
+// adaptation of https://bost.ocks.org/mike/shuffle/compare.html
+//
+// TODO
+// - vectorize the b-spline
+// - more shufflers
+// - measure reshuffle time
+// - swappable RNG
```

Comment:
Feel free to add this to TODO.txt, or add a built-in function `set_rnd_generator` or whatever.

Reply:
Lofted most out into `dev/TODO.txt`, and expanded on my own internal musings on where the PRNG stdlib should head...